core/modules/commands/makelibrary.js

```diff
@@ -35,8 +35,10 @@ Command.prototype.execute = function() {
 	var collectPlugins = function(folder) {
 			var pluginFolders = fs.readdirSync(folder);
 			for(var p=0; p<pluginFolders.length; p++) {
+				var currentFolder = path.resolve(folder,"./" + pluginFolders[p]);
+				if(!fs.statSync(currentFolder).isDirectory()) continue
```

Comment:
As noted elsewhere, our coding style is to always use braces for the bodies of "if" statements.

Also, we avoid using "continue" in the core code because it makes code hard to follow.

Finally, there is a built in function for checking if a path is a directory: see https://github.com/Jermolene/TiddlyWiki5/blob/master/core/modules/utils/filesystem.js#L142